kratos/includes/registry_item.h

```diff
@@ -18,7 +18,12 @@
 #include <string>
 #include <iostream>
 #include <unordered_map>
-#include <any>
+
+#if defined (__GNUC__) && __GNUC__ <= 8 && __GNUC_MINOR__ <= 3
+    #include <boost/any.hpp>
+#else
+    #include <any>
+#endif 
```

Comment:
@sunethwarna @pooyan-dadvand Not the cleanest solution in the world, but apparently is a bug in gcc (my suspicion is that it has something to do with the type_id().name(), but gcc bugtrack is horrendous and cannot find the exact bug report).

This should be removed when we move to dev-toolset-10/11

Reply:
Oh.... so the `boost/any` is superior than `std::any`.... It would be great if we can move to higher dev-toolset soon.... @pooyan-dadvand